src/terrain/routes/schemas/qms.clj

```diff
@@ -168,11 +168,12 @@
    (optional-key :status) (describe (maybe String) "The status of the request")})
 
 (defschema SubscriptionRequest
-  {(optional-key :username)  (describe String "The username to associate with the subscription")
-   (optional-key :plan_name) (describe String "The name of the plan to associate with the subscription")
-   :paid                     (describe Boolean "True if the user paid for the subscription")
-   (optional-key :periods)   (describe Integer "The number of subscription periods")
-   (optional-key :end_date)  (describe String "The end date of the subscription.")})
+  {(optional-key :username)   (describe String "The username to associate with the subscription")
+   (optional-key :plan_name)  (describe String "The name of the plan to associate with the subscription")
+   :paid                      (describe Boolean "True if the user paid for the subscription")
+   (optional-key :periods)    (describe Integer "The number of subscription periods")
+   (optional-key :start_date) (describe String "The start date of the subscfiption")
```

Comment:
Apparently my deal today is gonna be finding typos :joy: 

`subscfiption`

Reply:
Apparently, I'm very good at creating typos. 🤣 Thanks for catching that!